Intersect.Client.Core/Localization/Strings.cs

```diff
@@ -1285,6 +1276,24 @@ public partial struct Inventory
         public static LocalizedString Title = @"Inventory";
     }
 
+    public partial struct EntityContextMenu
+    {
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public static LocalizedString Friend = @"Befriend";
+
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public static LocalizedString Guild = @"Invite to Guild";
+
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public static LocalizedString Party = @"Invite to Party";
```

Comment:
Friend I think should be `Add Friend`

And can these 3 string names better match their contents?